sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,296 @@
+# September'22 release notes
+
+Release notes for commit range [`4043dda3..0f579bae`](https://github.com/intel/llvm/compare/4043dda3...0f579bae)
+
+**TODO**
+
+- commit 2ffd50a8cf8aefec0582da0ab5908c1665dcdbb6
```

Comment:
```suggestion
```

The commit makes changes related to `SYCL_PI_LEVEL_ZERO_USE_IMMEDIATE_COMMANDLISTS` environment variable, which is listed under [Debugging variables for Level Zero Plugin](https://github.com/intel/llvm/blob/sycl/sycl/doc/EnvironmentVariables.md#debugging-variables-for-level-zero-plugin). Therefore, I assume that we can omit from release notes. Tagging @rdeodhar to comment